pandas_gbq/gbq.py

```diff
@@ -2,11 +2,15 @@
 import logging
 import os
 import time
+import uuid
 import warnings
 from datetime import datetime
+from io import BytesIO
```

Comment:
I believe we need to use the `six` package for Python 2.7 support.

Reply:
I'll check. I had thought `BytesIO` was the same in both